src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
 // write code here
+
+const employeeList = document.querySelector('ul');
+
+// Convert the data-salary attribute from a string to a number
+
+function parseSalary(salarySring) {
```

Comment:
There is a typo in the function parameter name `salarySring`. It should be `salaryString` to correctly represent the parameter being passed.